### DISCUSSION TITLE
Search Across Organizations

### DISCUSSION THREAD

**Discussion by adrianjean** [craftcms/console#62] [Craft Console] [answered]
Would be great to be able to search for licences (CraftCMS, Plugins) across all organizations (checkbox: Search all Organizations). 

I love the ability to keep things organized by client, but then I lose the ability to track down licenses without going into each org one at a time and searching.

ps. Thank you SO much for creating Organizations. Brilliant.

**Comment by timkelty** [comment, marked as answer]
Global search is on our list!
Thanks for the FR. Stay tuned.
